import_data_into_db.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from sqlalchemy import Column, Float, Integer, String, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+Base = declarative_base()
```

Comment:
The model logic should be in a file called models.py

Reply:
Do you mean to move the Base variable and Animes class to a new .py file (models.py)?

Reply:
Yes please